src/Web/Sfa.Das.Sas.Shared.Components.Web/Sfa.Das.Sas.Shared.Components.Web.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
+  </ItemGroup>
+
+
+  <ItemGroup>
```

Comment:
Think this can be deleted?

Reply:
the file? its required for visual studio still!

Reply:
Sorry I meant the `FAT` folder in the `ItemGroup`

Reply:
ah! i've removed 👍